lib/node.js

```diff
@@ -780,6 +780,22 @@ declare class http$ClientRequest extends stream$Writable {
   setSocketKeepAlive(enable?: boolean, initialDelay?: number): void;
 }
 
+declare class http$ServerResponse extends stream$Writable {
+  addTrailers(headers: {[key: string] : string}): void;
+  finished: boolean;
+  getHeader(name: string): void;
+  headersSent: boolean;
+  removeHeader(name: string): void;
+  sendDate: boolean;
+  setHeader(name: string, value: string | Array<string>): void;
+  setTimeout(msecs: number, callback?: Function): http$ServerResponse;
```

Comment:
I was surprised by this return type... https://nodejs.org/api/http.html#http_response_settimeout_msecs_callback